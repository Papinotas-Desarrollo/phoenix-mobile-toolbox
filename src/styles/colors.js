const colors = {
  menuGrey: '#7D7D7D',
  bottomMenu: 'rgba(249, 249, 249, 0.9)',
  lightGrey: '#F7F7F7',
  black03: '#00000056',
  iconBlack: '#707070',
  papinotasBlue: '#4A90E2',
  backgroundLightBlue: '#4A90E220',
  papinotasOrange: '#FF9500',
  menuGray: '#8E8E93',
  lineGray: '#D1D1D6',
  buttons: {
    inactive: '#8E8E93',
    active: '#007AFF',
  },
  white: '#FFFFFF',
  black: '#000000',
  green: '#1C6900',
  red: '#FF0000',
  transparent: 'transparent',
  orange: '#f5a623',
  mediumGray: '#9a9a9a',
  blueFilter: '#7dafef',
  backgroundGrey: '#3690E9',
};

export default colors;
