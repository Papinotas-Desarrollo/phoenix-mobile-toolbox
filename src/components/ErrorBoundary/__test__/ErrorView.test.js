import * as React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { ErrorView } from '../';
import colors from '../../../styles/colors';

describe('Snapshot test', () => {
  it('With props snapshot', () => {
    const tree = renderer.create(<ErrorView />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Functionality test', () => {
  it('With props snapshot', () => {
    const tree = renderer.create(<ErrorView />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
