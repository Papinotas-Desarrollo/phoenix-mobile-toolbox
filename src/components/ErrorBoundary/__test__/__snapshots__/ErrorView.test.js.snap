// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test With props snapshot 1`] = `
Array [
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "#4A90E2",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Array [
          Object {
            "backgroundColor": "#4A90E2",
            "zIndex": 0,
          },
          0,
        ]
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#4A90E2",
            "elevation": 0,
            "flexDirection": "row",
            "height": 56,
            "marginTop": 0,
            "paddingHorizontal": 4,
          }
        }
      >
        <View
          style={
            Object {
              "width": 48,
            }
          }
        />
        <View
          accessible={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "flex": 1,
                "paddingHorizontal": 12,
              },
              Array [
                false,
                Object {
                  "alignItems": "center",
                },
                undefined,
              ],
            ]
          }
        >
          <Text
            accessibilityRole="header"
            accessibilityTraits="header"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "Helvetica Neue",
                  "textAlign": "left",
                  "writingDirection": "ltr",
                },
                Array [
                  Object {
                    "color": "#000000",
                    "fontFamily": "Helvetica Neue",
                  },
                  Object {
                    "fontSize": 17,
                  },
                  undefined,
                ],
              ]
            }
          >
            
          </Text>
        </View>
        <View
          accessibilityRole="button"
          accessible={true}
          hitSlop={
            Object {
              "bottom": 6,
              "left": 6,
              "right": 6,
              "top": 6,
            }
          }
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "overflow": "hidden",
              },
              Array [
                Object {
                  "alignItems": "center",
                  "borderRadius": 18,
                  "height": 36,
                  "justifyContent": "center",
                  "margin": 6,
                  "overflow": "visible",
                  "width": 36,
                },
                undefined,
                undefined,
              ],
            ]
          }
        >
          <View
            style={null}
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "justifyContent": "center",
                  },
                  Object {
                    "height": 24,
                    "width": 24,
                  },
                ]
              }
            >
              <View
                style={
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "transform": Array [
                      Object {
                        "rotate": "0deg",
                      },
                    ],
                  }
                }
              >
                <Text
                  accessibilityElementsHidden={true}
                  allowFontScaling={false}
                  importantForAccessibility="no-hide-descendants"
                  pointerEvents="none"
                  style={
                    Array [
                      Object {
                        "color": "#FFFFFF",
                        "fontSize": 24,
                      },
                      Array [
                        Object {
                          "transform": Array [
                            Object {
                              "scaleX": 1,
                            },
                          ],
                        },
                        Object {
                          "backgroundColor": "transparent",
                        },
                      ],
                      Object {
                        "fontFamily": "Material Icons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </RCTSafeAreaView>
    <RCTScrollView>
      <View>
        <View
          style={
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "#FFFFFF",
                  "fontSize": 250,
                },
                Object {
                  "transform": Array [
                    Object {
                      "rotate": "45deg",
                    },
                  ],
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontSize": 48,
                "fontWeight": "300",
                "paddingBottom": 16,
              }
            }
          >
            UPS!
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontSize": 20,
                "fontWeight": "400",
                "textAlign": "center",
                "width": 150,
              }
            }
          >
            Ocurrió un error en el sistema.
          </Text>
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "backgroundColor": "#4A90E2",
                "borderColor": "#FFFFFF",
                "borderRadius": 25,
                "borderWidth": 1,
                "marginHorizontal": 5,
                "marginTop": 75,
                "opacity": 1,
                "padding": 16,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Intenta de nuevo
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "backgroundColor": "#4A90E2",
                  "borderColor": "#FFFFFF",
                  "borderRadius": 25,
                  "borderWidth": 1,
                  "marginHorizontal": 5,
                  "marginTop": 75,
                  "opacity": 1,
                  "padding": 16,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#FFFFFF",
                      "fontSize": 25,
                    },
                    undefined,
                    Object {
                      "fontFamily": "Material Icons",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                
              </Text>
            </View>
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "backgroundColor": "#4A90E2",
                  "borderColor": "#FFFFFF",
                  "borderRadius": 25,
                  "borderWidth": 1,
                  "marginHorizontal": 5,
                  "marginTop": 75,
                  "opacity": 1,
                  "padding": 16,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#FFFFFF",
                      "fontSize": 25,
                    },
                    undefined,
                    Object {
                      "fontFamily": "Material Icons",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                
              </Text>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>,
  <Modal
    animationType="fade"
    hardwareAccelerated={false}
    onRequestClose={[Function]}
    onShow={[Function]}
    transparent={true}
    visible={false}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#00000080",
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderColor": "#FFFFFF",
              "borderRadius": 10,
              "borderWidth": 1,
              "justifyContent": "center",
              "paddingHorizontal": 20,
              "width": "80%",
            },
            Object {},
          ]
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "marginHorizontal": 10,
              "marginVertical": 10,
            }
          }
        >
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "backgroundColor": "#4A90E2",
                "borderColor": "#FFFFFF",
                "borderRadius": 25,
                "borderWidth": 1,
                "marginTop": 0,
                "opacity": 1,
                "padding": 16,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Opciones avanzadas
            </Text>
          </View>
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "backgroundColor": "#4A90E2",
                "borderColor": "#FFFFFF",
                "borderRadius": 25,
                "borderWidth": 1,
                "marginTop": 0,
                "opacity": 1,
                "padding": 16,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Ok
            </Text>
          </View>
        </View>
      </View>
    </View>
  </Modal>,
]
`;
